slides/reveal-js/technical_introduction_20240912/index.html

```diff
@@ -0,0 +1,343 @@
+                        </div>
+                        <div class="fragment" data-fragment-index="4">
+                            <p><strong>Schritt 7:</strong> Der Nutzer des PSP erkennt den Fehler durch die
+                                Empfangsbestätigung und korrigiert die Rezeptanforderung entsprechend.</p>
+                        </div>
+                        <div class="fragment" data-fragment-index="5">
+                            <p><strong>Schritt 8:</strong> Die korrigierte Rezeptanforderung wird erneut an das PVS des
+                                Arztes gesendet.</p>
+                        </div>
+                        <div class="fragment" data-fragment-index="6">
+                            <p><strong>Schritt 9:</strong> Das PVS verarbeitet die korrigierte Anforderung erfolgreich
+                                und
+                                sendet eine Bestätigung über die erfolgreiche Verarbeitung zurück an das PSP.</p>
+                        </div>
+                        <div class="fragment" data-fragment-index="7">
+                            <p><strong>Schritt 10:</strong> Die Rezeptanforderung und das ausgestellte Rezept werden
+                                ohne
+                                weiteres Eingreifen des Arztes an die Pflegeeinrichtung übermittelt.</p>
+                        </div>
+                    </div>
+                </section>
+
+
+            </section>
+
+
+            <section>
+                <h3>Entwicklung und Optimierung durch kollaborative Netzwerkinteraktion</h3>
+                <div style="font-size:0.75em; margin-top:0px; flex: 1; margin-left: 20px;">
+                    <p><strong>Entwicklung einer gemeinschaftlichen und qualitätsorientierten Umgebung:</strong></p>
+                    <p>Das Teilen von Adressen durch "First Mover" in Entwicklungsumgebungen erlaubt es
+                        Entwicklern, ihre Systeme durch Tests und Anpassungen kontinuierlich zu verbessern,
+                        was die Nachrichtenqualität im Netzwerk stetig erhöht.</p>
+                    <ul>
+                        <li><strong>Transparenz und Zugänglichkeit:</strong> Jede Nachricht im ATF-Netzwerk muss
+                            klar das sendende Softwaresystem ausweisen sowie Informationen darüber bereitstellen,
+                            wie die
+                            Entwickler dieses Systems erreicht werden können.</li>
+                        <li><strong>Qualitätssteigerung durch Feedback:</strong> Die Möglichkeit, direkt Feedback zu
+                            geben und zu erhalten, fördert die schnelle Behebung von Problemen und Unklarheiten und
+                            trägt
+                            so zur Qualitätssteigerung der gesamten Netzwerkkommunikation bei.</li>
+                        <li><strong>Kollaborative Optimierung:</strong> Durch die frühzeitige Integration und das
```

Comment:
Aktuell haben wir eine Seite, wo man sich als Hersteller eintragen lassen kann: https://github.com/gematik/spec-E-Rezept-ServiceRequest/discussions/61

Wollen wir noch aufnehmen, dass die gematik die Plattform zur Vernetzung bieten kann?

Reply:
Ja, guter Punkt.  Habe ich unter "Zentrale Fehlerkommunikation und Eskalation" in der Folie "Welches Problem löst das ATF?" aufgeführt.